shared/settings/notifications/render.tsx

```diff
@@ -94,19 +100,24 @@ const Notifications = (props: Props) =>
             </Kb.Text>{' '}
             and add an email address.
           </Kb.Text>
-          <Kb.Divider style={styles.divider} />
         </Kb.Box2>
       )}
+      <Kb.Divider style={styles.divider} />
       {(!Styles.isMobile || props.mobileHasPermissions) &&
       props.groups.app_push &&
       props.groups.app_push.settings ? (
-        <PhoneSection {...props} />
-      ) : Styles.isMobile /* TODO: display something if the user needs to enable push? */ ? null : (
-        <Kb.Box2 direction="vertical">
-          <Kb.Text type="Header">Phone notifications</Kb.Text>
-          <Kb.Text type="BodySmall">Install the Keybase app on your phone.</Kb.Text>
+        <>
+          <PhoneSection {...props} />
           <Kb.Divider style={styles.divider} />
-        </Kb.Box2>
+        </>
+      ) : (
```

Comment:
I think this set of parens isn't necessary.

Reply:
If I remove them prettier adds them back